app/vmbackup/README.md

```diff
@@ -167,6 +167,23 @@ See [this article](https://medium.com/@valyala/speeding-up-backups-for-big-time-
   -customS3Endpoint=https://s3-fips.us-gov-west-1.amazonaws.com
 ```
 
+* Usage with official [docker-compose](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/deployment/docker/docker-compose.yml) with  `vmsingle` instance:
```

Comment:
I'm unsure whether this will work as expected in docker-compose. The `vmbackup` exits after the backup is complete. This means that docker-compose will restart the `vmbackup` every time it finishes making the previous backup if I understand the logic behind `docker-compose`. This is not what users expect, since backups are usually made with the pre-defined interval instead of doing continuous backups. Probably, [vmbackupmanager](https://docs.victoriametrics.com/vmbackupmanager.html) should run instead of `vmbackup` here?

Reply:
Agree. Moreover, I think we shouldn't overcomplicate docker-compose env here. Its main purpose just to run VM locally to play around. The more components we add here, the more complicated and slow it becomes. Actually, I was against vmalert and vmagent in the current setup :-)

For users who seek examples and how-to-guides we need to provide the latter. It might be more useful and detailed than service in docker-compose.

Alternatively, we can provide multiple docker-compose manifests for different purposes:
1. prom + vm
2. vmagent + vm
3. vm + vmalert
4. vm + backuper

All from the list above can and should be converted into simple how-to-guides.